frontend/elements/src/core/popups/popups-template.ts

```diff
@@ -0,0 +1,89 @@
+import { MEDIA_UI } from '@utils/path';
+import { LitElement, html, css, customElement, property } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+
+
+export type Color = "peach" | "green" ;
+export type Size ="medium" | "large";
+
+
+@customElement('template-popups')
```

Comment:
needs to match the filename, e.g. `popups-template` instead of `template-popups`

However, there's a couple problems with that:

1. This element does not contain multiple popups, so it should just be popup
2. "template" has other meaning, here it seems to just be a filler word to mean "empty"

Suggestion: "popup-empty" or "popup-container"